packages/kbn-code-owners/src/file_code_owner.ts

```diff
@@ -65,14 +65,13 @@ export function getPathsWithOwnersReversed(): PathWithOwners[] {
 export function getCodeOwnersForFile(
   filePath: string,
   reversedCodeowners?: PathWithOwners[]
-): string | undefined {
+): { path: string; teams: string } | undefined {
```

Comment:
With that change we also need to update `packages/kbn-test/src/mocha/junit_report_generation.js` to not just include the whole return object but rather just `owners.teams`.

Reply:
There are a couple more places that need an update. [Here](https://github.com/elastic/kibana/blob/62c1333303624b6ddd9bacef04a78eab0c51aa09/packages/kbn-scout-reporting/src/reporting/playwright.ts#L63) and [here](https://github.com/elastic/kibana/blob/4feed672e3c890214213c95033729334310fedf6/packages/kbn-test/src/functional_test_runner/lib/mocha/reporter/scout_ftr_reporter.ts#L67).

Reply:
https://github.com/elastic/kibana/pull/202704/commits/e81017a9e529a0b844c0b50e2747ef5541bea337